packages/plugin-commands-env/src/env.ts

```diff
@@ -173,6 +191,45 @@ export async function handler (opts: NvmNodeCommandOptions, params: string[]) {
     return `Node.js ${nodeVersion} is removed
   ${versionDir}`
   }
+  case 'list':
+  case 'ls': {
+    if (!opts.global && !opts.cliOptions?.remote) {
```

Comment:
I am not sure the list command would work without the `--global` option. What would it list? Maybe it should be just `pnpm env list` and it would list all node.js versions on the system.

Reply:
It will list all node.js versions installed via `pnpm env use <version>`. Not all node.js versions on the system, it's hard to find the versions installed by other tools

Reply:
I will remove the global option and make it the default behavior

Reply:
`pnpm env use <version>` would still add the install node.js version to the central (global) location. It would just not make it  the default for the system.

Reply:
I don't distinguish whether global option is used in `pnpm env use <version>`, I just read the installing directory under PNPM_HOME